test/test_sparse.py

```diff
@@ -2093,6 +2093,10 @@ def test_pickle(self):
             sp_tensor_loaded = pickle.loads(serialized)
             self.assertEqual(sp_tensor, sp_tensor_loaded)
 
+    def test_div_by_sparse_error(self):
+
+        self.assertRaisesRegex(RuntimeError, 'Unsupported tensor layout', 
+                               lambda: torch.tensor(1., device=self.device) / torch.tensor(1., device=self.device).to_sparse())
```

Comment:
I don't think this is right. Shouldn't we be testing sparse / sparse division? (this is dense / sparse division right now).

Also, `torch.zeros(1, layout = torch.sparse, device=torch.device)` is a faster way to create sparse tensors.

Reply:
ah I must have pasted wrong. good to know about the `layout=` option

Reply:
Any idea what I'm doing wrong here?

```
In [22]: torch.ones(())
Out[22]: tensor(1.)

In [23]: torch.ones((), layout=torch.sparse)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-23-ed8e3fb1ad17> in <module>
----> 1 torch.ones((), layout=torch.sparse)

TypeError: ones() received an invalid combination of arguments - got (tuple, layout=module), but expected one of:
 * (tuple of ints size, tuple of names names, torch.dtype dtype, torch.layout layout, torch.device device, bool pin_memory, bool requires_grad)
 * (tuple of ints size, Tensor out, torch.dtype dtype, torch.layout layout, torch.device device, bool pin_memory, bool requires_grad)
```

Reply:
The name of the layout is actually `torch.sparse_coo` ...

Reply:
`full(...) is not implemented for sparse layout` ... `zeros` works but it seems a bit awkward where the test is division.